addons/website_sale_slides/static/src/js/slides_course_quiz.js

```diff
@@ -19,7 +23,48 @@ sAnimations.registry.websiteSlidesQuizNoFullscreen.include({
 
 Quiz.include({
     xmlDependencies: (Quiz.prototype.xmlDependencies || []).concat(
-        ["/website_sale_slides/static/src/xml/website_sale_slides_quiz.xml"]
-    )
+        ["/website_sale_slides/static/src/xml/channel_management.xml"]
```

Comment:
You should put the Quiz related override in the "website_sale_slides_quiz.xml" instead of re-grouping them into the channel_management one.